src/webview_provider/queryResultPanel.ts

```diff
@@ -312,6 +354,32 @@ export class QueryResultPanel extends AltimateWebviewProvider {
     this._panel!.onDidChangeVisibility(sendQueryPanelViewEvent);
   }
 
+  private setupTabWebviewHooks() {
+    this._tabPanel!.webview.onDidReceiveMessage(
```

Comment:
it will be better to reuse the existing onDidReceiveMessage handler, otherwise this tab will lose the other command handling. You should be using existing `setupWebviewHooks` so that all the commands will go through same flow. Otherwise it will be difficult to maintain multiple versions of the query results panel

Reply:
I already tried to do that but the webview of tab isn't available when only panel opens. So, we need to set it when it opens.